devel/cpptoml/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           boost 1.0
+
+github.setup        skystrife cpptoml 0.1.1 v
+checksums           rmd160  8471ec7641bccc67299cef3ddcf1b5f113fd6dbe \
+                    sha256  23af72468cfd4040984d46a0dd2a609538579c78ddc429d6b8fd7a10a6e24403 \
+                    size    47398
+
+categories          devel
+license             MIT
+
+maintainers         nomaintainer
+
+description         A header-only library for parsing TOML configuration files.
```

Comment:
if there are indeed only  header files in the package (haven't checked myself), you should probably add `installs_libs no` to indicate that

Reply:
Here's what `man 7 portfile` says about `installs_libs`:

> **installs_libs**
> 
> By default, it is assumed that ports may install libraries or headers
> that can be incorporated into their dependents. If this is not the case,
> set **installs_libs** to *no*. This means that this port’s dependents
> need not check that it is installed for the same architectures as them;
> that it is permissible to distribute binaries of the dependents even if
> their licenses conflict with the license of this port; and that updates
> to this port can never result in broken dynamic linking in its
> dependents. **  
> Type:** *optional* **  
> Default:** *none* **  
> Example:**
> 
> installs_libs no

So it looks like I should *not* use `installs_libs no`.